upgrading/topics/keycloak/changes-18_0_0.adoc

```diff
@@ -18,3 +18,41 @@ the migration. Consider these possible actions:
 insert into CLIENT_SCOPE_CLIENT (CLIENT_ID, SCOPE_ID, DEFAULT_SCOPE) select CLIENT.ID as CLIENT_ID, CLIENT_SCOPE.ID as SCOPE_ID, true as DEFAULT_SCOPE
 from CLIENT_SCOPE, CLIENT where CLIENT_SCOPE.REALM_ID='test' and CLIENT_SCOPE.NAME='acr' and CLIENT.REALM_ID='test' and CLIENT.PROTOCOL='openid-connect';
 ```
+
+= OpenID Connect Logout
+
+Previous versions of {project_name} had support for automatic logout of the user and redirecting to the application by opening logout endpoint URL like
+`https://auth-server/auth/realms/{realm-name}/protocol/openid-connect/logout?redirect_uri=encodedRedirectUri`. This was easy to use, but had potentially negative impact
+on performance and security. The new version has proper support for logout based on the OpenID Connect RP-Initiated Logout specification. The parameter `redirect_uri` is not
+supported anymore and user needs to confirm the logout. It is possible to omit the confirmation and do automatic redirect to the application when you include parameter `post_logout_redirect_uri`
+together with the parameter `id_token_hint` with the ID Token used for login.
+
+The existing deployments are affected as described below:
+
+- If your application directly uses links to logout endpoint with the `redirect_uri` parameter, you may be required to change this as described above.
+  It is recommeded to either remove `redirect_uri` parameter entirely or replace it with `id_token_hint` and `post_logout_redirect_uri` parameters.
+- If you use java adapters and your application does logout by call `httpServletRequest.logout()`, you are not affected because this call uses the backchannel variant of the logout endpoint
+  and that one was not changed.
+- If you use latest javascript adapter, you are also not affected. However if your application uses older version of javascript adapter, you are affected as this
+  adapter uses the variant of the logout endpoint with the deprecated `redirect_uri` parameter. In this case, you may need to upgrade to the latest version of the javascript adapter.
+
+There is backwards compatibility option, which allows that your application can still use the old format of the `redirect_uri` parameter. It is possible to enable it
```

Comment:
There is a backwards compatibility option, which allows your application to still use the old format of the `redirect_uri` parameter. You can enable it
at the server level by using this configuration when starting the Quarkus distribution: